modules/jax/src/main/ts/ephox/jax/core/Http.ts

```diff
@@ -140,12 +140,12 @@ const sendProgress = (init: HttpTypes.DownloadHttpRequest, loaded: number) => {
 
 const getMimeType = (headers: Headers) => Optional.from(headers.get('content-type')).map((value) => value.split(';')[0]);
 
-const fetchDownload = (init: HttpTypes.DownloadHttpRequest): FutureResult<Blob, HttpError> => FutureResult.nu((resolve) => {
+const fetchDownload = (init: HttpTypes.DownloadHttpRequest): FutureResult<Blob, HttpError<DataType.Blob>> => FutureResult.nu((resolve) => {
   const fail = (message: string, status: number) => {
-    resolve(Result.error({
+    resolve(Result.error<Blob, HttpError<DataType.Blob>>({
       message,
       status,
-      responseText: ''
+      responseText: new Blob()
```

Comment:
Not sure about this. Changed to `Blob` to avoid different return types for `fetchDownload` and `fallbackDownload` and end up with something like this:

`const download = (init: HttpTypes.DownloadHttpRequest): FutureResult<Blob, HttpError<DataType.Blob>> | FutureResult<Blob, HttpError<DataType.Text>>`

Reply:
That looks great to me, good thinking